8-CSS-Position-and-Transform/Part-1/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css" />
+  <link rel="stylesheet" href="./css/all.min.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet" />
+  <title>Hostel</title>
+</head>
+
+<body>
+  <img class="background" src="./img/bg.png" alt="" />
```

Comment:
this is our header and all its element must be wrapped with one section or div or <header>